Templates/make_RedMine_SGs.tmplt.json

```diff
@@ -52,15 +53,52 @@
         "VpcId": { "Ref": "TargetVPC" }
       }
     },
-    "UpdateNasSg": {
+    "UpdateNasSgRpcbindTcp": {
       "DependsOn": "NasSg",
       "Type": "AWS::EC2::SecurityGroupIngress",
       "Properties": {
+        "Description": "Exceptions for rpcbind over TCP",
         "GroupId": { "Ref": "NasSg" },
         "SourceSecurityGroupId": { "Ref": "NasSg" },
         "IpProtocol": "tcp",
-        "FromPort": "0",
-        "ToPort": "65535"
+        "FromPort": "111",
+        "ToPort": "111"
+      }
+    },
+    "UpdateNasSgRpcbindUdp": {
+      "DependsOn": "NasSg",
+      "Type": "AWS::EC2::SecurityGroupIngress",
+      "Properties": {
+        "Description": "Exceptions for rpcbind over UDP",
+        "GroupId": { "Ref": "NasSg" },
+        "SourceSecurityGroupId": { "Ref": "NasSg" },
```

Comment:
Is `NasSg` attached to both the redmine EC2 instance and the EFS volume?

Reply:
The EC2 instance has all three SGs. Each of the SGs lassos the EC2 and a given service (i.e., EC2/EFS; EC2/ELB; EC2/RDS)

Reply:
Got it, just making sure, cuz otherwise this wouldn't work. 😆